keyboards/canary/canary60rgb/v2/keyboard.json

```
@@ -0,0 +1,222 @@
+            {"flags": 4, "matrix": [1, 6], "x": 104, "y": 16},
+            {"flags": 4, "matrix": [1, 7], "x": 120, "y": 16},
+            {"flags": 4, "matrix": [1, 8], "x": 136, "y": 16},
+            {"flags": 4, "matrix": [1, 9], "x": 152, "y": 16},
+            {"flags": 4, "matrix": [1, 10], "x": 168, "y": 16},
+            {"flags": 4, "matrix": [1, 11], "x": 184, "y": 16},
+            {"flags": 4, "matrix": [1, 12], "x": 200, "y": 16},
+            {"flags": 4, "matrix": [1, 13], "x": 220, "y": 16},
+
+            {"flags": 1, "matrix": [2, 0], "x": 6, "y": 32},
+            {"flags": 4, "matrix": [2, 2], "x": 28, "y": 32},
+            {"flags": 4, "matrix": [2, 3], "x": 44, "y": 32},
+            {"flags": 4, "matrix": [2, 4], "x": 60, "y": 32},
+            {"flags": 4, "matrix": [2, 5], "x": 76, "y": 32},
+            {"flags": 4, "matrix": [2, 6], "x": 92, "y": 32},
+            {"flags": 4, "matrix": [2, 7], "x": 108, "y": 32},
+            {"flags": 4, "matrix": [2, 8], "x": 124, "y": 32},
+            {"flags": 4, "matrix": [2, 9], "x": 140, "y": 32},
+            {"flags": 4, "matrix": [2, 10], "x": 156, "y": 32},
+            {"flags": 4, "matrix": [2, 11], "x": 172, "y": 32},
+            {"flags": 4, "matrix": [2, 12], "x": 188, "y": 32},
+            {"flags": 1, "matrix": [2, 13], "x": 214, "y": 32},
+
+            {"flags": 1, "matrix": [3, 0], "x": 9, "y": 48},
+            {"flags": 4, "matrix": [3, 2], "x": 36, "y": 48},
+            {"flags": 4, "matrix": [3, 3], "x": 52, "y": 48},
+            {"flags": 4, "matrix": [3, 4], "x": 68, "y": 48},
+            {"flags": 4, "matrix": [3, 5], "x": 84, "y": 48},
+            {"flags": 4, "matrix": [3, 6], "x": 100, "y": 48},
+            {"flags": 4, "matrix": [3, 7], "x": 116, "y": 48},
+            {"flags": 4, "matrix": [3, 8], "x": 132, "y": 48},
+            {"flags": 4, "matrix": [3, 9], "x": 148, "y": 48},
+            {"flags": 4, "matrix": [3, 10], "x": 164, "y": 48},
+            {"flags": 1, "matrix": [3, 11], "x": 186, "y": 48},
+            {"flags": 1, "matrix": [3, 12], "x": 208, "y": 48},
+            {"flags": 1, "matrix": [3, 13], "x": 224, "y": 48},
+            
+            {"flags": 1, "matrix": [4, 0], "x": 2, "y": 64},
+            {"flags": 1, "matrix": [4, 2], "x": 22, "y": 64},
+            {"flags": 1, "matrix": [4, 3], "x": 42, "y": 64},
+            {"flags": 4, "matrix": [4, 4], "x": 102, "y": 64},
+            {"flags": 1, "matrix": [4, 5], "x": 160, "y": 64},
+            {"flags": 1, "matrix": [4, 8], "x": 176, "y": 64},
+            {"flags": 1, "matrix": [4, 11], "x": 192, "y": 64},
+            {"flags": 1, "matrix": [4, 12], "x": 208, "y": 64},
+            {"flags": 1, "matrix": [4, 13], "x": 224, "y": 64}
+        ],
+        "led_process_limit": 4,
+        "led_flush_limit": 26,
+        "max_brightness": 180,
+        "sleep": true
+    },
+    "layouts": {
+        "LAYOUT": {
```

Consideration taken before accepting this suggestion, as it would also require changes within `v1` and `keymaps` directories as seen in #23497
```suggestion
    "community_layouts": ["60_ansi_arrow"],
    "layouts": {
        "LAYOUT_60_ansi_arrow": {
```